rhv_checks.py

```diff
@@ -338,6 +340,54 @@ def check_vms_distributed_hosts(system, warn=5, crit=10):
         sys.exit(3)
 
 
+def check_hosted_engine_status(system, **kwargs):
+    """ Check the status of all the host's Hosted Engine Status."""
+    okay, critical, unknown, all_items = [], [], [], []
+    command = "hosted-engine --vm-status --json"
+    hosts = system.api.system_service().hosts_service().list()
+    output = ssh_command(hosts[0].name, password, command)
+    # Output is in the format of List of JSON removing newline.
+    if len(output) != 0:
+        result = output[0].replace("\\n", "").replace("\n", "")
+        host_data_dict = json.loads(result)
+        # Iterating value from Dict --> Dict --> Value.
+        for host_index, host_detail in host_data_dict.items():
+            if host_index == 'global_maintenance':
+                continue
+            try:
+                health = host_detail['engine-status']['health']
+                if health == "good":
+                    okay.append((host_detail["hostname"], health))
+                elif health == "bad":
+                    critical.append((host_detail["hostname"], health))
+            except KeyError:
+                health = "unknown"
+                unknown.append((host_detail["hostname"], health))
+            all_items.append((host_detail["hostname"], health))
+    else:
+        print("Check hosted-engine command is available or not.")
```

Comment:
Please add a call to logger as well to log this.

Reply:
Logger is added.